lib/util/auth.js

```diff
@@ -1,10 +1,13 @@
-const TYPE = require('../../types/mcdev.d');
-const Util = require('./util');
-const File = require('./file');
-const SDK = require('sfmc-sdk');
-const Conf = require('conf');
-const credentialStore = new Conf({ configName: 'sessions', clearInvalidConfig: true });
-// const currentMID = null;
+import TYPE from '../../types/mcdev.d.js';
+import { Util } from './util.js';
+import File from './file.js';
+import SDK from 'sfmc-sdk';
+import Conf from 'conf';
+const credentialStore = new Conf({
+    projectName: 'mcdev',
+    configName: 'sessions',
+    clearInvalidConfig: true,
+});
```

Comment:
generally bumping versions as many relate to es6 updates - in this case because of how the file system is handing, the projectname from package.json cannot be ascertained SO you need to declare it.